### PR TITLE
Feature/multi database (Petronas)

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**

### CODE REVIEW

File: vantage6/cli/node.py [hunk 1]
```diff
@@ -356,16 +358,26 @@ def cli_node_start(name, config, environment, system_folders, image, keep,
...
+            debug('  - non file-based database added')
+            env[f'{LABEL}_DATABASE_URI'] = uri
+        else:
+            debug('  - file-based database added')
+            env[f'{LABEL}_DATABASE_URI'] = f'/mnt/{label}.csv'
+            mounts.append((f'/mnt/{label}.csv', str(uri)))
+
+        # FIXME legacy to support < 2.1.3 can be removed from 3+
+        if label == 'default':
+            env['DATABASE_URI'] = '/mnt/default.csv'
```
Comment (bartvanb) [inline vantage6/cli/node.py:382]: If we still want to remove this from version 3, it is better to delete it now (before people start using it in v3)

**Review by bartvanb — COMMENTED** [review]
Just one small comment. Haven't tested the functionality so if we want to include this in v3 soon we should do so (but I think we'd do that anyway right?).

File: vantage6/cli/node.py (hunk 1 above)
Comment (frankcorneliusmartin) [reply vantage6/cli/node.py:382]: Yes, thats a good idea. Multi-database is not fully implemented yet into v3. Lets do that first before removing this part.

**Comment by frankcorneliusmartin** [comment]
This needs to be merged for: IKNL/vantage6-main#175

**Review by bartvanb — APPROVED** [review]

**Review by bartvanb — APPROVED** [review]
No further comments

**Comment by bartvanb** [comment]
Just resolved some merge conflicts with DEV3

**Pull request merged**